webapp/tests/karma/ts/services/xml-forms.service.spec.ts

```diff
@@ -1112,6 +1127,9 @@ describe('XmlForms service', () => {
         })
         .catch(err => {
           expect(err.message).to.equal(`The form "${internalId}" doesn't have an xform attachment`);
+          expect(error.callCount).to.equal(1);
+          expect(error.args[0][0]).to.equal('Error in XMLFormService getDocAndFormAttachment : ');
+          expect(error.args[0][1]).to.equal(`The form "${internalId}" doesn't have an xform attachment`);
```

Comment:
Could you please add a test where the `getByView` function throws an error and assert over what is thrown and what is logged?

Reply:
I think I did, please the comments below

Reply:
So, a test where the stub to get the view is something like: 
```
dbQuery.rejects({ an: 'error' });
```

Reply:
ok, will do for `getById` as well.

Reply:
ok